### ISSUE TITLE
[RAML Verifier] Verification weird failure. 

### ISSUE THREAD

**Issue by arielsegura** [phoenixnap/springmvc-raml-plugin#70]
I took this RAML with its controller from some tests:

```
#%RAML 0.8
title: myapi
baseUri: /v1
version: 1

/base:
  /1:
    /{2}:
      get:
      /3:
        get:
          responses:
            200:
              body:
                application/json:
                  example: |
                    {
                      "status" : "UP"
                    }
        post:
```

```
package bla;


import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/v1/base/1/{2}")
public class VerifierTestController {

    public void unannotatedMethod() {

    }

    @RequestMapping( method = { RequestMethod.GET })
    public String simpleMethodAllHttpMethods() {
        return null;
    }

    @RequestMapping(value = "/3", method = { RequestMethod.GET, RequestMethod.POST })
    public String simpleGetMethod() {
        return null;
    }

}
```
and pasted them into a new Spring Boot App. When I checked the RAML by invoking maven-goal verify-springmvc-api-docs, it says:
```
[INFO] Performing Resource Existence Checks
[DEBUG] Checking for any missing resources in implementation
[ERROR] Expected resource missing: /base in SOURCE
[DEBUG] Checking for any missing resources in RAML
[WARNING] Expected resource missing: /v1 in CONTRACT
[DEBUG] Checking resource /base
[DEBUG] Checking resource /base
[DEBUG] Checking resource /1
[DEBUG] Collection Resource identified: /1
[DEBUG] Checking resource /1
[DEBUG] Checking resource /{2}
[DEBUG] Checking resource /{2}
[DEBUG] Checking Action: GET
[DEBUG] Checking Action: GET
[DEBUG] Checking Action: GET
[DEBUG] Checking Action: GET
[DEBUG] Checking resource /3
[DEBUG] Checking resource /3
[DEBUG] Checking Action: GET
[DEBUG] Checking Action: GET
[DEBUG] Checking Action: GET
[DEBUG] Checking Action: GET
[DEBUG] Checking Action: POST
[DEBUG] Checking Action: POST
[DEBUG] Checking Action: POST
[DEBUG] Checking Action: POST
[WARNING] Issue [severity=WARNING, location=CONTRACT, type=MISSING, description=Missing Resource., ramlLocation=/v1]
[WARNING] Issue [severity=WARNING, location=CONTRACT, type=STYLE, description=Collections of Resources should be Pluralised in the URL, ramlLocation=/base/1]
[WARNING] Issue [severity=WARNING, location=CONTRACT, type=STYLE, description=GET Verb should define NOT_FOUND (404) response, ramlLocation=GET /base/1/{2}]
[WARNING] Issue [severity=WARNING, location=CONTRACT, type=STYLE, description=GET Verb should define NOT_FOUND (404) response, ramlLocation=GET /base/1/{2}/3]
[ERROR] Issue [severity=ERROR, location=SOURCE, type=MISSING, description=Missing Resource., ramlLocation=/base]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE

```
Which is wrong since the raml and its controller seems to be totally valid.

**Comment by kurtpa** [comment]
Off the cuff i think you need to use the uriPrefixToIgnore configuration parameter to ignore the "v1" which is listed in your base uri. Since it is not trivial to know where an api is deployed, you need to use this to sync if you are putting part of the url in the baseUri. 

(optional) A Portion of the URL of the implemented RAML that will be ignored when verifying.

The pluralisation errors are caused by having "1" as a resource name which is not really a good name and cannot be singularised/pluralised. You can also disable this style check through configuration

**Comment by kurtpa** [comment]
Any updates on this?

**Comment by arielsegura** [comment]
Hi there, sorry for the delay. I guess it was an usage issue. After some thoughts I decided to modify the raml so the API version (V1) is included as a resource rather than as a baseUri, which is wrong. API versioning is a controversial topic though, I couldn't get to any conclusion so far :) 
Will close this issue

**Comment by kurtpa** [comment]
no probleme :) the other issue you opened is fixed and will be released to maven central shortly (version 0.7.2)
